### PR TITLE
Update BUILDING.md

### PR DESCRIPTION
**Pull request by AlaaFkirin**
Update the Ubuntu Linux building instructions for first-time users.

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -14,7 +16,7 @@ All of them are listed in the following installation command:
 
 ```
 Linux:
-sudo apt-get update && sudo apt-get install build-essential git gperf libgmp-dev cmake bison curl flex gcc-multilib linux-libc-dev libboost-all-dev libtinfo-dev ninja-build python3-setuptools unzip wget
+sudo apt-get update && sudo apt-get install build-essential git gperf libgmp-dev cmake bison curl flex gcc-multilib linux-libc-dev libboost-all-dev libtinfo-dev ninja-build python3-setuptools unzip wget python3-pip openjdk-8-jre
```
Comment (rafaelsamenezes) [inline BUILDING.md:20]: Why did you need `openjdk-8-jre` package? Are you sure about it?

**Review by rafaelsamenezes — CHANGES_REQUESTED** [review]
Minor comments

File: BUILDING.md (hunk 1 above)
Comment (AlaaFkirin) [reply BUILDING.md:20]: This is because of an error that happened while setting up CVC4 solver, and I preferred to add `openjdk-8-jre`  package on the "Installing Dependencies" step rather than on "Setting Up CVC4" step.
image

Comment (rafaelsamenezes) [reply BUILDING.md:20]: Yeah I agree, nice finding!
